news/189.documentation

```diff
@@ -0,0 +1 @@
+
```

Comment:
Remove the file, or add an actual changelog entry.

Reply:
Done in b29a2d07e0900ec1ea8509340cd491101fc8b0e4